specificatie/BRK-Bevragen/genereervariant/openapi.yaml

```diff
@@ -3841,10 +3847,10 @@ components:
       description: Hieraan wordt een Kadasterstuk of een Aangebodenstuk gekoppeld.
         Een stuk is een brondocument dat aanleiding geeft tot een wijziging van de
         gegevens in een basisregistratie.
-    PortefeuilleId:
+    PortefeuilleID:
       type: object
       properties:
-        akrRegistercode:
+        AKRregistercode:
```

Comment:
Waarom is AKR hier met hoofdletters en akrStukdeel1 niet?

Reply:
in imKad heet het "akrRegistercode". Die past mooi in de camelCase conventie. Ben dus ook benieuwd naar de reden om dit te wijzigen.